LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any customer obtaining a copy
```

Comment:
```suggestion
Permission is hereby granted, free of charge, to any person obtaining a copy
```
Should we change the MIT license back to person? :P I highly doubt we need to change the MIT License as well.